modules/transformer-kinesis/src/test/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/kinesis/processing/TestApplication.scala

```diff
@@ -36,10 +36,12 @@ object TestApplication {
       parsed <- ShredderCliConfig.Stream.loadConfigFrom[IO]("Streaming transformer", "Test app")(args).value
       res <- parsed match {
         case Right(cliConfig) =>
-         Resources.mk[IO](cliConfig.igluConfig, queueFromDeferred(forCompletionMessage))
+         Resources.mk[IO](cliConfig.igluConfig, queueFromDeferred(forCompletionMessage), cliConfig.config.monitoring.metrics)
           .use { resources =>
             logger[IO].info(s"Starting RDB Shredder with ${cliConfig.config} config") *>
               Processing.runWindowed[IO](windowedRecords, resources, cliConfig.config)
+                .compile
```

Comment:
What do you think about enabling stdout reporter in tests?  Or even better... providing custom `Reporter` implementation  to test metrics reporting? I'm not saying we should do that, may be overkill, just thinking out load :)